layouts/shortcodes/dbm-mongodb-agent-beta-install-linux.md

```diff
@@ -0,0 +1,13 @@
+To install the beta version of the Datadog Agent on a Linux host, run the following command. Replace `<DD_API_KEY>` with your [Datadog API key][1].
+
+```shell
+# Override the following environment variables
+export DD_REPO_URL=datad0g.com
```

Comment:
I haven't seen this repo URL before, just wanted to confirm that it's correct? `datad0g.com` works in the browser, I just haven't seen it used like this.

Reply:
this is specific to install the beta agent version because the mongodb dbm private beta has not been included in the prod agent release.

Reply:
Got it, thank you!